aimmo-game/tests/functional/test_pickups_apply.py

```diff
@@ -0,0 +1,42 @@
+import random
+from unittest import TestCase
+
+from simulation import map_generator
+from tests.test_simulation.dummy_avatar import DummyAvatarManager, MoveEastDummy
+from tests.test_simulation.dummy_avatar import MoveEastDummy
+from simulation.turn_manager import SequentialTurnManager
+from simulation.location import Location
+from simulation.pickups import HealthPickup
+
+
+SETTINGS = {
+    'START_HEIGHT': 5,
+    'START_WIDTH': 5,
+    'OBSTACLE_RATIO': 0,
+}
+
+
+class TestGame(object):
+    def __init__(self, settings=SETTINGS):
+        random.seed(0)
+        self.generator = map_generator.Main(SETTINGS)
+        self.avatar_manager = DummyAvatarManager([MoveEastDummy])
+        self.game_state = self.generator.get_game_state(self.avatar_manager)
+        self.turn_manager = SequentialTurnManager(game_state=self.game_state, end_turn_callback=lambda: None, completion_url='')
+
+
+class TestEffects(TestCase):
```

Comment:
More reflective of file name, we might want to consider changing `test_pickups_apply.py` to `test_health_pickups_and_effects_applied.py` etc.